"Method","Name","Error","Occurrences"
"GET","/api/profiles/test","('Connection aborted.', error(54, 'Connection reset by peer'))",325
"GET","/api/articles/un-titulo-creado-por-pedro-nkjgdh/comments","('Connection aborted.', error(54, 'Connection reset by peer'))",344
"GET","/api/articles/un-titulo-creado-por-pedro-nkjgdh","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/articles/un-titulo-creado-por-pedro-nkjgdh",503
"GET","/api/articles","('Connection aborted.', error(54, 'Connection reset by peer'))",315
"GET","/api/articles","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/articles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10a3c9c50>: Failed to establish a new connection: [Errno 60] Operation timed out',))",123
"GET","/api/profiles/test","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/profiles/test",485
"GET","/api/tags","('Connection aborted.', error(54, 'Connection reset by peer'))",308
"PUT","/api/user","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/user",452
"GET","/api/articles","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/articles",454
"GET","/api/articles/un-titulo-creado-por-pedro-nkjgdh/comments","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/articles/un-titulo-creado-por-pedro-nkjgdh/comments (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10812d950>: Failed to establish a new connection: [Errno 60] Operation timed out',))",89
"GET","/api/tags","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/tags",467
"GET","/api/user","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/user (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d0cdd0>: Failed to establish a new connection: [Errno 60] Operation timed out',))",89
"POST","/api/users/login","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/users/login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10a76dd90>: Failed to establish a new connection: [Errno 60] Operation timed out',))",231
"GET","/api/articles/un-titulo-creado-por-pedro-nkjgdh","('Connection aborted.', error(54, 'Connection reset by peer'))",317
"PUT","/api/user","('Connection aborted.', error(54, 'Connection reset by peer'))",349
"POST","/api/users/login","('Connection aborted.', error(54, 'Connection reset by peer'))",393
"GET","/api/profiles/test","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/profiles/test (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109bb8650>: Failed to establish a new connection: [Errno 60] Operation timed out',))",90
"PUT","/api/user","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/user (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1083c8f50>: Failed to establish a new connection: [Errno 60] Operation timed out',))",92
"GET","/api/user","('Connection aborted.', error(54, 'Connection reset by peer'))",313
"GET","/api/tags","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/tags (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1093f6f90>: Failed to establish a new connection: [Errno 60] Operation timed out',))",105
"GET","/api/articles/un-titulo-creado-por-pedro-nkjgdh/comments","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/articles/un-titulo-creado-por-pedro-nkjgdh/comments",482
"GET","/api/articles/feed?limit=10","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/articles/feed?limit=10 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109464e10>: Failed to establish a new connection: [Errno 60] Operation timed out',))",79
"GET","/api/articles/un-titulo-creado-por-pedro-nkjgdh","HTTPConnectionPool(host='10.6.128.158', port=80): Max retries exceeded with url: /api/articles/un-titulo-creado-por-pedro-nkjgdh (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095f9050>: Failed to establish a new connection: [Errno 60] Operation timed out',))",107
"GET","/api/articles/feed?limit=10","('Connection aborted.', error(54, 'Connection reset by peer'))",339
"GET","/api/user","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/user",442
"POST","/api/users/login","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/users/login",225
"GET","/api/articles/feed?limit=10","500 Server Error: Internal Server Error for url: http://10.6.128.158/api/articles/feed?limit=10",450